testbed/HIL_Firmware/PDU_HIL/PDU_HIL/processCAN.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/queue.h"
+#include "freertos/semphr.h"
+#include "esp_err.h"
+#include "esp_log.h"
+#include "driver/twai.h"
+#include "userInit.h"
+#include "canReceive.h"
+#include "processCAN.h"
+#include "digitalPot.h"
+
+static uint32_t dByte0 = 0;
+static uint32_t dByte1 = 0;
+static uint32_t dByte2 = 0;
+
+static uint32_t dByte0_mask = 0xFFFF00;
```

Comment:
I might be wrong, but the 00 is already at the start, because the way I set it up is to pass the physical resistance number into the CAN message so we can just send the resistance value we need